.github/workflows/shellcheck.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           exclude-path: |
             tests/**
             examples/**/test.sh
+            templates/**
```

Comment:
Hm, `shellcheck` still complains. I guess this should be `tmt/templates/**`?

Reply:
You are right. It should be fixed now.